packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -200,8 +199,16 @@ class TransferEngine<
     }
   }
 
+  validateTransferOptions() {
+    if (!VALID_STRATEGIES.includes(this.options.strategy)) {
+      throw new Error('Invalid stategy ' + this.options.strategy);
+    }
+  }
+
   async transfer(): Promise<ITransferResults<S, D>> {
     try {
+      this.validateTransferOptions();
```

Comment:
I didn't add this, so I'm assuming I made a mistake while merging and it's been deleted or refactored in features/deits. Can anyone confirm?